tests/conftest.py

```diff
@@ -1,6 +1,6 @@
 import builtins
 import pytest
-from tools import DummyShell, sp
+from tests.tools import DummyShell, sp
```

Comment:
I am a pytest n00b.  I originally changed it because I was getting test failures in master.  But I no longer get the error (even in my branch), so I've reverted the change.